src/topology/category/CompHaus.lean

```diff
@@ -159,3 +159,41 @@ has_limits_of_has_limits_creates_limits CompHaus_to_Top
 
 instance CompHaus.has_colimits : limits.has_colimits CompHaus :=
 has_colimits_of_reflective CompHaus_to_Top
+
+namespace CompHaus
+
+/--
```

Comment:
It's not very important, but personally I prefer to put these docstring delimiters on the same line as the docstring (just like with `{` and `}`). Maybe `/-!` for module docstrings is an exception. It's a trivial way to save vertical space and squeeze more info on my screen.
But currently mathlib isn't very consistent about this. Maybe we should normalize this somewhere?

Reply:
I agree, but in this case 
`/-- An explicit limit cone for a functor `F : J ⥤ CompHaus`, defined in terms of `Top.limit_cone`. -/`
is one character over the limit.

Reply:
Sure, but you could still put it on 2 lines instead of 4, right?